server/sonar-web/src/main/js/apps/custom-measures/components/List.tsx

```diff
@@ -0,0 +1,125 @@
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import { sortBy } from 'lodash';
+import DeleteButton from './DeleteButton';
+import EditButton from './EditButton';
+import { CustomMeasure } from '../../../app/types';
+import ActionsDropdown, {
+  ActionsDropdownDivider
+} from '../../../components/controls/ActionsDropdown';
+import { translate } from '../../../helpers/l10n';
+import Tooltip from '../../../components/controls/Tooltip';
+import { formatMeasure } from '../../../helpers/measures';
+import DateFormatter from '../../../components/intl/DateFormatter';
+
+interface Props {
+  measures: CustomMeasure[];
+  onDelete: (measureId: string) => Promise<void>;
+  onEdit: (data: { description: string; id: string; value: string }) => Promise<void>;
+}
+
+export default function List({ measures, onDelete, onEdit }: Props) {
+  return (
+    <div className="boxed-group boxed-group-inner" id="custom-measures-list">
+      {measures.length > 0 ? (
+        <table className="data zebra zebra-hover">
+          <thead>
+            <tr>
+              <th>{translate('custom_measures.metric')}</th>
+              <th>{translate('value')}</th>
+              <th>{translate('description')}</th>
+              <th>{translate('date')}</th>
+              <th />
+            </tr>
+          </thead>
+          <tbody>
+            {sortBy(measures, measure => measure.metric.name.toLowerCase()).map(measure => (
+              <tr data-metric={measure.metric.key} key={measure.id}>
+                <td className="nowrap">
+                  <div>
+                    <span className="js-custom-measure-metric-name">{measure.metric.name}</span>
+                    {measure.pending && (
+                      <Tooltip overlay={translate('custom_measures.pending_tooltip')}>
+                        <span className="js-custom-measure-pending badge badge-warning spacer-left">
+                          {translate('custom_measures.pending')}
+                        </span>
+                      </Tooltip>
+                    )}
+                  </div>
+                  <span className="js-custom-measure-domain note">{measure.metric.domain}</span>
+                </td>
+
+                <td className="nowrap">
+                  <strong className="js-custom-measure-value">
+                    {formatMeasure(measure.value, measure.metric.type)}
+                  </strong>
+                </td>
+
+                <td>
+                  <span className="js-custom-measure-description">{measure.description}</span>
+                </td>
+
+                <td>
+                  <MeasureDate measure={measure} /> {translate('by_')}{' '}
+                  <span className="js-custom-measure-user">{measure.user.name}</span>
```

Comment:
I'd use `little-spacer-left` instead of `{' '}`

Reply:
I think it looks like a sentence this way:

![image](https://user-images.githubusercontent.com/2317341/36107019-280bf00a-1019-11e8-92b6-db6a043b8b62.png)